packages/fxa-content-server/server/lib/sentry.js

```diff
@@ -89,16 +89,29 @@ if (config.get('sentry.dsn')) {
  */
 function tryCaptureValidationError(err) {
   try {
-    const errorDetails =
-      err?.details instanceof Map && err?.details?.get('body');
+    // Try to get error details. This might not be present.
+    let errorDetails = null;
+    if (err != null && err.details instanceof Map) {
+      errorDetails = err.details.get('body');
+    }
 
     if (errorDetails) {
       const message = `${errorDetails}`;
       const validationError = errorDetails.details.reduce((a, v) => {
-        return {
-          ...a,
-          [v?.path?.join('.')]: `reason: ${v.type} - ${v.message}`,
-        };
+        // Try to get the key for the field that failed validation and update
+        // the error state
+        if (v && v.path && v.path.join) {
```

Comment:
Not sure if something like `if (Array.isArray(v?.path)) {…}` is any better/clearer here.

Reply:
Per the issue filed, I can't use optional chaining syntax, ie `?.` is not an option, because the js parser, acorn, that get text uses doesn't support it. 

I do like your suggestion of using the more explicit Array.isArray check though 👍 . So I'll adjust accordingly.